### DISCUSSION TITLE
[RELEASE] Fight Club

### DISCUSSION THREAD

**Discussion by Paindeluxe** [nzp-team/nzportable#523] [Map Releases]
_"This is Fight Club."_

> These are the Rules
> 1. Don't talk about Fight Club
> 2. Don't use Guns.
> 3. Do not jump out of the Ring.
> 4. Send these Zombies to hell.

This is a small map with next to no space to train the zombies.
Your only Weapon is your trusty pocket knife and a Bowie knife...

_How long will you last?_

# Preview

![fightclub](https://github.com/nzp-team/nzportable/assets/104042185/6f20971b-daa4-4c4e-a1b8-b6241b3155c9)
![fightclub1](https://github.com/nzp-team/nzportable/assets/104042185/c82269c5-c2ed-41d9-9d0b-82123161d48a)

If you want to edit the map or see how I made this, click [THIS](https://github.com/nzp-team/nzportable/files/13628694/FightClubSource.zip) to download the uncompiled version of the map. If not, click the [DOWNLOAD](https://github.com/nzp-team/nzportable/files/13628693/FightClub.zip) button down below

# [DOWNLOAD](https://github.com/nzp-team/nzportable/files/13628693/FightClub.zip)

**Comment by MotoLegacy** [comment]
Hi, for this to quality for release standards make sure the gfx, models,etc. are in the root directory of the zip archive. Also, unregulated but still requested, consider making the menu preview pictures low-ram (PSP, 3DS) complaint. Thanks.

**Comment by Paindeluxe** [comment]
Little observation I made, this map does not boot on 3ds, expect other platforms except the pc to not play this either until next week.

Apparently you cannot use the light_spot entity to well, light up the map.

**Comment by MotoLegacy** [comment]
`light_spot` is valid and works. Check _Nacht der Untoten_ for a visual example. It's used everywhere there.

**Reply by Paindeluxe** [reply]
Though I recieved a _pitch error?

**Reply by MotoLegacy** [reply]
Yes, that's junk. Lightmaps are baked on compile. VHLT/ZHLT supports `light_spot`, so everything does.

**Reply by Paindeluxe** [reply]
![IMG_0056](https://github.com/nzp-team/nzportable/assets/104042185/afeaabe8-6377-49cf-b787-d1e8cd7e8bf5)

This is the error log that I get

**Reply by MotoLegacy** [reply]
It's complaining about your `func_ending`. Did you assign it a model?

**Reply by MotoLegacy** [reply]
Just checked, no you don't: `"model" ""`. Give it a model.

**Reply by M0THRAK** [reply]
Hey, do you mind if I de-compile and remake this map, me and my friends have been loving it and I wanna add more.

**Reply by DerpedCrusader** [reply]
They provide the .map in the post if im not mistaken, just above the main download

**Reply by M0THRAK** [reply]
I really spent all that time... anyways thanks lmao, this will save me a ton of time.

**Comment by MotoLegacy** [comment]
Finally getting to test this, your crew names aren't compliant with the documented limit of 16 characters. You can read that spec here: https://docs.nzp.gay/mapping/player-spawns#specifying-character-names

**Comment by MotoLegacy** [comment]
You're able to dolphin dive outside of the ring.

![fte-20231210175829-0](https://github.com/nzp-team/nzportable/assets/26030250/3d7da71a-4999-4a42-90f0-80de4c32702b)

**Reply by Paindeluxe** [reply]
That was intended to be a placeholder easter egg, and would not be kept for V2 as I don’t want to fill the faces outside of the ring

**Reply by Paindeluxe** [reply]
There is some „dialog“ if you walked towards the staminup machine and the buyable ending.

Now that I explained it I don’t wanna keep this in anymore

(Yes the dialog has a member from the audience say the wrong rule, there was no time for me to adjust that)

**Comment by Paindeluxe** [comment]
My version of the .map lost 2 layers (Hard and Easy Difficulty) so screw it I am abandoning and disowning this map

**Comment by MotoLegacy** [comment]
Closing it makes it inaccessible for others. Dont be a prude.

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as the download is not hosted on archive.org

**Comment by M0THRAK** [comment]
Hey, I've remade this map if anyone's interested in playing, you can download it **[here](https://archive.org/download/fight-club_202606/FightClub%2B.zip)**

**Reply by ProfessorZIRE** [reply]
remake!

**Reply by nonMat** [reply]
<img width="1920" height="1080" alt="immagine" src="https://github.com/user-attachments/assets/26c7d711-a97f-48c6-88ed-ecf835172600" />
I would have preffered a non barricade version but this was still fun!
Actually I was plannign on doing a remake of this but you preceeded me XD

**Comment by nonMat** [comment]
<img width="1920" height="1080" alt="immagine" src="https://github.com/user-attachments/assets/f358d976-c00e-44ea-8743-3f53c46de8bf" />
Fun! The Hard version got me really pumped!
